config/v0.4.0.json

```diff
@@ -0,0 +1,197 @@
+        "spec",
+        "test",
+        "exception",
+        "todo",
+        "implication"
+      ]
+    },
+    "HtmlReport": {
+      "type": "object",
+      "properties": {
+        "blob-link": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/TemplatedString"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "enabled": {
+          "default": true,
+          "type": "boolean"
+        },
+        "issue-link": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/TemplatedString"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "path": {
+          "default": ".duvet/reports/report.html",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "JsonReport": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "default": false,
+          "type": "boolean"
+        },
+        "path": {
+          "default": ".duvet/reports/report.json",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Report": {
+      "type": "object",
+      "properties": {
+        "html": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/HtmlReport"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "json": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/JsonReport"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        }
+      },
+      "additionalProperties": false
+    },
+    "Requirement": {
+      "type": "object",
+      "required": [
+        "pattern"
+      ],
+      "properties": {
+        "pattern": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Source": {
+      "type": "object",
+      "required": [
+        "pattern"
+      ],
+      "properties": {
+        "comment-style": {
+          "$ref": "#/definitions/CommentStyle"
+        },
+        "pattern": {
+          "type": "string"
+        },
+        "type": {
+          "$ref": "#/definitions/DefaultType"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Specification": {
+      "type": "object",
+      "properties": {
+        "format": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/SpecificationFormat"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "source": {
+          "default": "",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "SpecificationFormat": {
+      "type": "string",
+      "enum": [
+        "ietf",
+        "markdown"
+      ]
+    },
+    "TemplatedString": {
+      "type": "string"
+    }
+  }
+}
```

Comment:
should these have a newline at the end?

Reply:
I dont feel very strongly, its more just the way GitHub highlights it that bothers me :-)